rules/google-workspace/google_workspace_admin_role_deletion.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/11/17"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects when a custom admin role is deleted. An adversary may delete a custom admin role in order to impact the
```

Comment:
I usually stay away from shortening words like admin, or legit, but in this case I think its fine.

Reply:
I used the shortened terms, as that's what Google uses in their documentation and UI. If you're happy with it, I'll go ahead and merge them 👍

Reply:
If Brent is happy, I am happy